filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilder.scala

```diff
@@ -64,14 +66,20 @@ class BlobPathBuilder(credential: AzureSasCredential, container: String, endpoin
   val fileSystemConfig: Map[String, Object] = Map((AzureFileSystem.AZURE_STORAGE_SAS_TOKEN_CREDENTIAL, credential),
                                                   (AzureFileSystem.AZURE_STORAGE_FILE_STORES, container))
 
+  def retrieveFilesystem(): Try[FileSystem] = {
```

Comment:
It would be good to pass a URI into this function rather than having it access the value on the class and create it multiple times. Total nitpicking, though, no need to hold merging up for that, just something to think about next time we're in here.